playbooks/roles/setup-cicd/tasks/cicd.yml

```diff
@@ -0,0 +1,63 @@
+---
+
+- name: Checkout devsecops-workshop repo
+  git:
+    repo: "https://github.com/epe105/devsecops-workshop.git"
+    dest: "/home/{{ ansible_user }}/devsecops-workshop"
+    version: "master"
+    force: yes
+
+- name: Copy script for caching CI/CD images across nodes
+  copy:
+    src: "../files/cache_docker_images.yml"
+    dest: "/home/{{ ansible_user }}/cache_docker_images.yml"
+    mode: 0644
+
+- name: Copy script for batch provisioning init variables
+  template:
+    src: "../files/provision-batch-init.sh"
+    dest: "/home/{{ ansible_user }}/devsecops-workshop/scripts/provision-batch-init.sh"
+    mode: 0775
+
+- name: Copy script for quay provisioning init variables
+  template:
+    src: "../files/provision-quay.sh"
+    dest: "/home/{{ ansible_user }}/devsecops-workshop/scripts/quay/provision-quay.sh"
+    mode: 0775
+
+- name: Copy script for CI/CD installation
+  template:
+    src: "../files/install-cicd.sh"
+    dest: "/home/{{ ansible_user }}/install-cicd.sh"
+    mode: 0775
+
+- name: Provide details to check CI/CD installation
+  debug:
+    msg: |
+         Check file '/home/{{ ansible_user }}/install-cicd.log' on '{{ openshift_public_hostname }}'
+         To access ec2 instance 'ssh -i {{ ec2_key_file }} ec2-user@{{ openshift_public_hostname }}'
+
+- name: Run CI/CD installation script
+  shell: ./install-cicd.sh |& tee install-cicd.log
+  args:
+   chdir: "/home/{{ ansible_user }}"
+
+- name: Update docker options
+  shell: "sed -i '/OPTIONS=.*/c\\OPTIONS=\"--selinux-enabled --signature-verification=False --insecure-registry docker-registry-default.apps.{{ openshift_public_hostname }}\"' /etc/sysconfig/docker"
+  become: true
+
+- name: Allow group to use oscap-docker without sudo password
+  lineinfile:
+    dest: /etc/sudoers
+    line: "%{{ generic_group | default('autobot') }} ALL=NOPASSWD: /usr/bin/oscap-docker"
+    validate: '/usr/sbin/visudo -cf %s'
+  become: true
+  when: install_oscap
```

Comment:
this conditional wasn't in the original main.yml. However, added this so that if oscap isn't installed, this task isn't executed